app/src/main/java/com/myfitnesstracker/ui/main/MainViewModel.kt

```diff
@@ -74,42 +74,42 @@ class MainViewModel : ViewModel() {
       fetchNutritionSearchResults("pho")
    }
 
-   internal var bmis:MutableLiveData<ArrayList<BMI>>
+   internal var bmis : MutableLiveData<ArrayList<BMI>>
```

Comment:
Added spaces before and after the semicolon.
According to: https://kotlinlang.org/docs/reference/coding-conventions.html